elements/pfe-dropdown/src/pfe-dropdown-item.scss

```diff
@@ -18,7 +18,8 @@ $LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
   width: 100%;
   padding: pfe-local(spacing--vertical) pfe-local(spacing--horizontal);
   color: pfe-local(Color);
-  font-size: pfe-local(FontSize);
+  //font-size: pfe-local(FontSize);
```

Comment:
Not sure we want to remove font-size declaration here - I liked your approach in pfe-cta.scss

Reply:
You're right, I went and added them back & missed this one, thanks!